docs/layouts/partials/earlier-version-warning.html

```diff
@@ -8,15 +9,17 @@
     {{ $previewVersion = printf "(%s)" .Site.Params.yb.preview_version }}
   {{ end }}
 
-  {{ if eq ($.Scratch.Get "docVersion") "stable" }}
-  <div class="admonition warning">
-    <p class="admonition-title">Attention</p>
-    <p>This page documents a stable (production) version. For testing and development with the latest features, use <a href="{{ $previewUrl }}">the preview {{ $previewVersion }} version.</a></p>
-  </div>
-  {{ else }}
-  <div class="admonition warning">
-    <p class="admonition-title">Attention</p>
-    <p>This page documents an earlier version. <a href="{{ $previewUrl }}">Go to the preview {{ $previewVersion }} version.</a></p>
-  </div>
-  {{ end }}
-{{ end }}
\ No newline at end of file
+  {{ range .Site.Data.currentVersions.dbVersions }}
+    {{- if or (eq $docVersion .series) (eq $docVersion .alias)  -}}
+      <div class="admonition warning">
+        {{- if and (eq .isStable true) (eq .isSTS true) -}}
```

Comment:
This logic should be "if isStable is true AND (if isSTS is true OR isLTS is true)"

Reply:
And perhaps we can simplify to just "if isSTS is true OR isLTS is true".